decorators/withAuthSync/withAuthSync.js

```diff
@@ -26,12 +36,6 @@ const withAuthSync = WrappedComponent =>
       window.localStorage.removeItem('logout');
     }
 
-    syncLogout = event => {
-      if (event.key === 'logout') {
-        Router.push('/login');
-      }
-    };
-
```

Comment:
Why did this get changed?

Reply:
ESLint:

<img width="616" alt="Screen Shot 2022-06-09 at 12 33 22 PM" src="https://user-images.githubusercontent.com/523330/172929672-015b4810-764f-4c25-94ef-30c11ce3695f.png">

We can the lint rule, would be better

Reply:
You have lint rules activated that are not actually activated in this codebase. Maybe it stems from that your dependencies somehow auto updated? We check lint rules on master and - as you can see - this file is not erroring.

Reply:
There is no lockfile change in this PR